config/software/ruby.rb

```diff
@@ -72,6 +72,9 @@
   env['SOLIBS'] = "-lm -lc"
   # need to use GNU m4, default m4 doesn't work
   env['M4'] = "/opt/freeware/bin/m4"
+when "solaris2"
+  env['CC'] = "/usr/sfw/bin/gcc -static-libgcc"
```

Comment:
Nope - this is explicitly overriding to use the 'old' gcc. Default gcc at this point will from with...flags will end up being the omnibus-build-essential gcc 4.9